.gitignore

```diff
@@ -1,4 +1,3 @@
 **/__pycache__/
 .pytest_cache/
 .hermes/
-.coverage
```

Comment:
Removed, as this is tooling-specific config that we cannot assume the user uses.